lib/api/authn/force-user.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('../../debug').authentication
+
+/**
+ * Enforces the `--force-user` server flag, hardcoding a webid for all requests,
+ * for testing purposes.
+ */
+function initialize (app, argv) {
+  const forceUserId = argv.forceUser
+  app.use('/', (req, res, next) => {
+    debug(`Identified user (override): ${forceUserId}`)
+    req.session.userId = forceUserId
+    res.set('User', forceUserId)
```

Comment:
That's right, but it was still in the `forceUser` code I moved here. Maybe this is useful for faking a TLS authentication? (After all, the client needs _some_ way to find out who the logged-in user is.)